KsApi/mutations/inputs/PostCommentInput.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct PostCommentInput: GraphMutationInput {
+  let body: String
+
+  /// This is base64 encoded ID of the Project-{projectId}}
+  let commentableId: String
+
+  /// If the comment is on the project level, this is nil. For replies, this is the ID of the comment.
```

Comment:
Is it more useful to leave this comment here or at the call site of the mutation (in the view model) ?

Reply:
Think this is good, Xcode should show it during one of those rare autosuggestion moments.